src/main/scala/model/FinalResult.scala

```diff
@@ -0,0 +1,5 @@
+package model
+
+import command.Move
+
+class FinalResult(val userMove: Move, val computerMove: Move, val gameResult: GameResult)
```

Comment:
c'è un motivo particolare per non aver usato una case class qui?

Reply:
sostanzialmente questa classe è stata pensata principalmente come contenirore del risultato di gioco, non utilizzata in nessuna computazione successiva perciò ho pensato che il case potesse essere omesso in questo caso.

Reply:
è una data class, quindi in generale mi sembra sensato usare una case class in questo senso